plant/plants/api/serializers.py

```diff
@@ -44,18 +55,20 @@ class Meta:
             "owner",
             "staff",
             "staff_ids",
+            "family_id",
+            "family",
         ]
 
     def create(self, validated_data):
-        staff = validated_data.pop("staff_ids")
         instance = super().create(validated_data)
-        instance.staff.set(staff)
+        validated_data["family"] = validated_data.pop("family_id")
```

Comment:
Correct me if I am wrong, but this should be executed before L63 as in current form value for family will be blank